### ISSUE TITLE
ARM64 Support

### ISSUE THREAD

**Issue by james10174** [scaleway/docker-machine-driver-scaleway#65]
Any idea when I can use docker-machine-driver-scaleway on a ARM64 servers? I assume currently I cannot do this currently as I tried

```
docker-machine create -d scaleway --scaleway-commercial-type=ARM64-2GB \
  --scaleway-token=$TOKEN --scaleway-organization=$ACCESS_KEY \
  --scaleway-name="cloud-scaleway-arm64-1" --engine-install-url="http://bit.ly/1sf3j8V" \
  cloud-scaleway-arm64

Running pre-create checks...
Creating machine...
(cloud-scaleway-arm64) Creating SSH key...
(cloud-scaleway-arm64) Creating server...
Error creating machine: Error in driver during machine creation: ARM64-2GB wrong commercial type
```

**Comment by xbogdan** [comment]
Same issue. Any updates ?

**Comment by karser** [comment]
I have the same issue with X64-15GB https://github.com/scaleway/docker-machine-driver-scaleway/issues/77
I guess it supports only VC1 and C2.

**Comment by izissise** [comment]
This is fixed in version 1.6
Thanks for reporting this issue
